### DISCUSSION TITLE
Text Snippet Management

### DISCUSSION THREAD

**Discussion by ghost** [kovidgoyal/kitty#6631] [Ideas]
It seems Kitty has the facilities to manage text snippets, similar to [the-way](https://github.com/out-of-cheese-error/the-way). Snippet terminal integration, as well as using snippets on TUI editors that don't support snippets, would be a useful feature. A native solution would, in my opinion, would be better than integrating send_text with a CLI snippet manager.

**Comment by kovidgoyal** [comment]
No kitty has no such facilities. Indeed, this is not possible to implement robustly, since the terminal can have no idea what pressing a key will do in the currently running program.
